test/component/nginx-app-protect/monitoring/testData/events-out/xml_violation_url_data.json

```diff
@@ -39,10 +39,18 @@
                 "Name": "VIOL_JSON_MALFORMED",
                 "Context": "url",
                 "ContextData": {
-                    "Name": "",
+                    "Name": "uri",
                     "Value": "/"
                 }
+            },
+            {
+                "Name": "VIOL_URL_CONTENT_TYPE",
+                "Context": "uri",
+                "ContextData": {
+                    "Name": "$",
+                    "Value": "actual header value: beni. matched header value: beni"
```

Comment:
question: it seems like we have a name containing the description at the time, for ex:
` "Name": "Header length: 42, exceeds Header length limit: 10",`

and, here we have the value containing the description:
`"Value": "actual header value: beni. matched header value: beni"`

Should we try to be consistent here?

Reply:
Applied 👍